CONTRIBUTING.md

```diff
@@ -137,12 +137,18 @@ improperly referenced, and the commit is not signed off by the author.
 
 ### Install a git commit message hook script to validate local commits
 
+We have a script that attempts to enforce some of these commit guidelines. You
+can install it by following instructions similar to the below. 
+
 ```
 cd omr_repo_dir
 cp scripts/commit-msg .git/hooks
 chmod +x .git/hooks/commit-msg
 ```
 
+Be sure to update the script on occasion, as it will be grow and adapt as our
+guidelines grow and adapt.
```

Comment:
This sentence is awkwardly written.  Perhaps:  "Be sure to update your version of the script occasionally as it may evolve as our commit guidelines grow and adapt."